learning_journal/views/views.py

```diff
@@ -0,0 +1,57 @@
+from pyramid.view import view_config
+from pyramid.httpexceptions import HTTPNotFound, HTTPFound
+from sqlalchemy.exc import DBAPIError
+from ..models import Entry
+
+
+@view_config(route_name="home", renderer="../templates/index.jinja2")
+def list_view(request):
+    """View home and list of journals."""
+    entries = request.dbsession.query(Entry).order_by(Entry.id.desc()) # order by date
+    return {"entries": entries}
+
+
+@view_config(route_name="edit", renderer="../templates/edit.jinja2")
+def edit_view(request):
+    """View the edit view."""
+    get_id = request.matchdict['id']
+    entry = request.dbsession.query(Entry).filter(Entry.id == get_id).first()
+    if request.method == "POST":
+        entry.title = request.params.get('title')
+        entry.body = request.params.get('body')
```

Comment:
do you want to do any validation or sanity checking on the provided title and body before you update your entry?